src/templates/assets/javascripts/browser/request/index.ts

```diff
@@ -85,7 +85,12 @@ export function request(
     // Handle download progress
     if (typeof options?.progress$ !== "undefined") {
       req.addEventListener("progress", event => {
-        options.progress$!.next((event.loaded / event.total) * 100)
+        if(event.lengthComputable){
```

Comment:
Please add whitespace around the braces ☺️

Reply:
Done.